Package.swift

```diff
@@ -39,6 +39,22 @@ let package = Package(
       name: "FirebaseAppDistribution-Beta",
       targets: ["FirebaseAppDistributionTarget"]
     ),
+    .library(
```

Comment:
In our meeting, we stated that the new library products weren't needed - but I'm not sure about that.  Maybe we need to add a little tool to find/insert products into the Package.swift for development?

Reply:
Happy to remove them for now - LMK.

Reply:
Perhaps, comment them out for now? @ryanwilson Any thoughts here?

Reply:
Hmm, yeah commenting them out sounds good for now.